Configuration/DataProcessing/python/Impl/trackingnoccc.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+"""
+_trackingnoccc_
+
+Scenario supporting proton collisions
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class trackingnoccc(pp):
+    def __init__(self):
+        pp.__init__(self)
+        self.recoSeq=':reconstruction_trackingOnly'
+        self.cbSc='pp'
+        self.promptCustoms  += [ 'RecoTracker/Configuration/customiseForRunIInoCCC.customiseForRunIInoCCC' ]
+        self.expressCustoms += [ 'RecoTracker/Configuration/customiseForRunIInoCCC.customiseForRunIInoCCC' ]
+
+    """
+    _trackingnoccc_
+
+    Implement configuration building for data processing for proton
+    collision data taking without CCC in the tracking sequence
+
+    """
+    def promptReco(self, globalTag, **args):
+        """
+        _promptReco_
+
+        Proton collision data taking prompt reco
+
+        """
+
+#        if not 'skims' in args:
```

Comment:
is this needed?
please clean up commented out parts or add some comments why these may be needed for the future